src/js/utils/ServiceConfigUtil.js

```diff
@@ -0,0 +1,70 @@
+import HostUtil from '../utils/HostUtil';
+import Networking from '../constants/Networking';
+
+const serviceAddressKey = 'Service Address';
+
+function buildHostName(id, port) {
+  let hostname = HostUtil.stringToHostname(id);
+  return `${hostname}${Networking.L4LB_ADDRESS}:${port}`;
+}
+
+function defaultCreateLink(contents) {
+  return contents;
+}
+
+function hasVIPLabel(portDefinition) {
+  return portDefinition.labels &&
+    Object.keys(portDefinition.labels).find(function (key) {
+      return /^VIP_[0-9]+$/.test(key);
```

Comment:
We're checking on `VIP_` including underscore, but in a fixture we don't have underscores (different PR because original code has been split) https://github.com/dcos/dcos-ui/pull/1119/files#diff-df117a9057b1e083308b664ffae7c922R248
Not sure what's the source of truth.